blueprints/video_discovery/scripts/kb/extract/extract_credits.py

```diff
@@ -1,13 +1,13 @@
 import logging
 import luigi
 import sys
-from video_task import VideoDataProcessingTask  # noqa: F401
-from libs.tasks import crawl_urls, request_api  # noqa: F401
-from utils import load_plain_json  # noqa: F401
 
 from .commons import API_KEY
 
 sys.path.append('..')
+from video_task import VideoDataProcessingTask  # noqa: F401,E402
```

Comment:
You can try:
```
from ..video_task import VideoDataProcessingTask  # noqa: F401
from ..utils import load_json  # noqa: F401
from ..libs.tasks import ReadLocalDir  # noqa: F401

Reply:
Can you make relative imports when its not a package?

Reply:
I think they are already doing relative import in line 5:
`from .commons import API_KEY`

Reply:
That is not possible with the current layout. The extract folder is being read as a subpackage with the init, but the kb folder doesn't have the same. Hence the need to have explicit sys.path appends and full imports! Merging this :)